# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: .cargo/config.in
# Commit: 634412ce2943
# Full Hash: 634412ce29430557eec6d0c0313f1908e61ad13d
# Author: Matthew Gregan <kinetik@flim.org>
# Date: 2021-08-11 03:46:10
# Regressor Bug: 1724141
# File Overlap Count: 6
# Description:
#   Bug 1724141 - Update audioipc to 8bb1a227.  r=chunmin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D121826
# ==============================================================================

diff -r 722284bed619 -r 634412ce2943 .cargo/config.in
--- a/.cargo/config.in	Tue Aug 10 20:38:51 2021 +0000
+++ b/.cargo/config.in	Tue Aug 10 20:52:03 2021 +0000
@@ -40,7 +40,7 @@
 [source."https://github.com/mozilla/audioipc-2"]
 git = "https://github.com/mozilla/audioipc-2"
 replace-with = "vendored-sources"
-rev = "7537bfadad2e981577eb75e4f13662fc517e1a09"
+rev = "8bb1a227fbaa5677458bcd876162b65307df38c2"
 
 [source."https://github.com/mozilla/application-services"]
 git = "https://github.com/mozilla/application-services"